include/experimental/__p0009_bits/extents.hpp

```diff
@@ -287,5 +295,19 @@ class extents
 
 };
 
+// CT: Initial implementation for dextents proposed in: P2299
```

Comment:
I implemented this since I use this as storage for strides. But I put it into detail namespace for now.